docs/index.md

```diff
@@ -8,6 +15,3 @@ To learn more about the notify service please visit [here](https://github.com/wa
 This plugin is used to send notifications on triggers like `on_task`, `on_start`, `on_failure`, `on_demand`
 during and post teflo execution. For more information on how to setup triggers for notification
 please visit [here](https://teflo.readthedocs.io/en/latest/users/definitions/notifications.html#triggers).
```

Comment:
Please add this:

"To know more about teflo please visit here <teflo's read the docs link>"

Reply:
done